### PR TITLE
prodigal: import from homebrew/science

### PR DESCRIPTION
**Pull request by sjackman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Migrate from Homebrew/science.

### CODE REVIEW

File: Formula/prodigal.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Prodigal < Formula
+  desc "Microbial gene prediction"
+  homepage "http://prodigal.ornl.gov/"
```
Comment (fxcoudert) [inline Formula/prodigal.rb]: `The URL http://prodigal.ornl.gov/ is not reachable`

File: Formula/prodigal.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/hyattpd/Prodigal/archive/v2.6.3.tar.gz"
+  sha256 "89094ad4bff5a8a8732d899f31cec350f5a4c27bcbdd12663f87c9d1f0ec599f"
+  head "https://github.com/hyattpd/Prodigal.git"
+
+  def install
+    system "make", "install", "INSTALLDIR=#{bin}"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/prodigal -v 2>&1")
```
Comment (fxcoudert) [inline Formula/prodigal.rb]: Can we test it on a simple input file?

**Review by fxcoudert — DISMISSED** [review]

File: Formula/prodigal.rb (hunk 2 above)
Comment (sjackman) [reply Formula/prodigal.rb]: I've added a test.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
thanks @sjackman!

**Comment by sjackman** [comment]
Thanks for the fix, and thanks for merging!

**Pull request closed without merging**